recipes/pynisher/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pynisher" %}
+{% set version = "0.6.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e4e1d9366fc4ca60b4b2354b6d12e65600600a8c7bf4392c84f2f4ff4abc85ff
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - docutils >=0.3
+    - psutil
+    - python >=3.6
+    - setuptools
+
+test:
+  imports:
+    - pynisher
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/automl/pynisher
+  summary: A small Python library to limit the resources used by a function by executing it inside a subprocess.
+  description: |
+    The pynisher is a little module intended to limit a functions resources. It starts a new process, sets the desired limits,
+    and executes the function inside it. In the end, it returns the function return value. If, for any reason,
+    the function call is not successful, None is returned.
+  license: MIT
+  license_file: LICENSE
+  dev_url: https://github.com/automl/pynisher
+
+extra:
+  recipe-maintainers:
+    - BastianZim
+    - mfeurer
```

Comment:
@mfeurer, are you happy to be a maintainer here? 🙂

Reply:
Already checked 😄 https://github.com/conda-forge/staged-recipes/pull/13861#issuecomment-772618036

Reply:
Oops 🤦‍♂️ Sorry 😬